scripts/gui/microbe_hud.mjs

```diff
@@ -481,27 +575,194 @@ function hideWinText(){
     document.getElementById("winContainer").style.display = "none";
 }
 
+function onCompressPanelClicked(panelToChange) {
+
+    // Variables needed for changes
+    let barToChange = "";
+    let titleToChange = "";
+    let valuesToChange = "";
+    let h = "";
+
+    // Determine different values because right now
+    // The two panels are  different in layout
+    if(panelToChange.getAttribute("id") == "compoundsPanel") {
```

Comment:
So this function actually works on one of two possible targets? I think that it would be way more clear if this was made into a parameter. So this would take an object that has properties like:
```
{
    panelToChange: "...",
    compressedBackgroundImage: "...",
    barToChange = "...",
    titleToChange = "...",
    valuesToChange = "...",
    offsetAdjustment: 92,
}
```